xml_inscriptions/bib_only/MA.Bos.MFA.G.84.44.xml

```diff
@@ -35,44 +36,131 @@
+                        century B.C.</date>
+                     <placeName ref="asia.asia_minor">Assos</placeName>
+                  </origin>
+                  <provenance type="given" subtype="">
+                     <p> gift of the <orgName>Archaeological Institute of America</orgName> to <orgName>MFA</orgName>, <date when="1884">January 15,1884</date></p>
+                  </provenance>
+                  <provenance type="discovered" subtype="">
+                     <p> From <placeName>Assos</placeName> (Behramkale, Turkey); on the Acropolis, north of the temple,
+                        near the surface of the ground. <date when="1882">May 10th, 1882</date>: excavated by the
+                        <orgName>Archaeological Institute of America</orgName></p>
+                  </provenance>
+               </history>
             </msDesc>
             <listBibl>
                <bibl>
                   <ptr type="rest-of-bibl" target="#Assos"/>
                   <biblScope unit="item">3</biblScope>
                </bibl>
+               <bibl>
+                  <ref type="external"
+                     target="https://collections.mfa.org/objects/199431/inscribed-fragment-with-list-of-public-measures-and-weights?ctx=70811cd2-1063-4aff-8bb9-ab6400de9364&amp;idx=0"
+                     >MFA web entry</ref>
+               </bibl>
             </listBibl>
          </sourceDesc>
       </fileDesc>
       <encodingDesc>
          <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
-                      href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
             <xi:fallback>
                <p>Taxonomies for US Epigraphy controlled values</p>
             </xi:fallback>
          </xi:include>
       </encodingDesc>
       <profileDesc/>
       <revisionDesc>
-         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009
+            EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-07" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.84.44.jpg"/>
+      </surface>
+   </facsimile>
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+            <ab> <supplied reason="lost">τ</supplied>ὰ σκεύεά ἐσσι δαμόσια ἐπὶ <lb/>
```

Comment:
In transcriptions, every line, even the first, has to start with an <lb>. Also, make sure to number the lines (i.e. `<lb n="1">`, etc.).

Reply:
Sorry if I'm not reading this correctly but what are you saying that every line have to start with? "an"? a period? a space?

Reply:
Sorry, the code snippet wasn't displaying properly; it should be fine now. Basically, any time a line begins, there needs to be an `<lb>`, and you should number all of them, even if there's only one.